style_editor.html

```diff
@@ -10018,16 +10077,117 @@
             id="ROTATE_BUTTON" title="Toggles animation of the preview blade above."
             onclick="ClickRotate()">
             <i class="fa-solid fa-arrows-rotate"></i> Rotate</button>
-          <input
-            type=button id=INHILT_BUTTON name=inhilt  value="Inhilt"
-            title="Simulates In-Hilt LED blades, (no addressable pixel effects)."
-            onclick="ClickInhilt()" >
-          <input
-            type=button id=SLOW_BUTTON name=inhilt value="Slow"
-            title="Provides &quot;snapshots&quot; of rapidly moving things to see what they look like without fast movement. *Note* Graflex hilt model probably won't be able to render with this on."
-            onclick="ClickSlow()" >
```

Comment:
We can get rid of ClickSlow and ClickInhilt, right?

Reply:
Yes they are gone. the buttons are in settings panel and all settings checkboxes and saves can be handled with handleClick().

Reply:
I don't see any delete lines for those functions though.

Reply:
My mistake. deleted now.